metadata.json

```diff
@@ -1,15 +1,12 @@
 {
-  "name": "puppetlabs-corosync",
+  "name": "puppet-corosync",
   "version": "0.7.0",
```

Comment:
other communities have solved this problem… this… fear…

http://hauptversionsnummernerhoehungs.angst.software